CHANGES.md

```diff
@@ -1,3 +1,11 @@
+# Version 0.14.1
+
+## Bugs
+- Fixed scheduled Cloud Functions execution with cron-job by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/3245
+- Fixed missing runtime icons by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/3234
+- Fixed Google OAuth by @Meldiron in https://github.com/appwrite/appwrite/pull/3236
+- Fixed allowing `localhost` as the main domain by @Meldiron in https://github.com/appwrite/appwrite/pull/3237
```

Comment:
More like `Fixed certificate generation when hostname set to 'localhost'`

You could have localhost. But then no domain would be main, so you didn't get certificate generated automatically.